js/about/preferences.js

```diff
@@ -311,6 +315,27 @@ class BitcoinDashboard extends ImmutableComponent {
       </div>
     }
   }
+  get exchangePanel () {
+    const exchangeInfo = this.props.ledgerData.get('exchangeInfo')
+    const url = exchangeInfo.get('exchangeURL')
+    const name = exchangeInfo.get('exchangeName')
```

Comment:
this causes the `Add funds` panel to throw an error and not appear at all if `exchangeInfo` is undefined. better to use `this.props.ledgerData.getIn(['exchangeInfo', propertyname])`